balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -71,6 +71,24 @@ public void run() {
         }
     }
 
+    /**
+     * Closing the Metric Consumer is effectively a delegation method that allows the Metric Consumers
+     * internal resources to close and clean up (If necessary).
```

Comment:
This is pretty much what all .close() methods do. You don't even have to write this if you are feeling lazy.